action_id: trunk-whoami
date: 2024-11-17 15:58:50
run: trunk whoami
run_from: ""
trigger: schedule 24h+1h
result:
  exit_code: 1
  out: ""
  err: Exception occurred while executing 'trunk-whoami'. See .trunk\out\S7yBv.yaml for details.
  error_path: C:\Users\holna\Pylearning\goit-cs-hw-02\.trunk\out\S7yBv.yaml
timestamp: 1731859130
full_timestamp:
  nanos: 112806000
  seconds: 1731859130
trunk_version: 1.22.8
